community/radare2.hpkg

```diff
@@ -0,0 +1,105 @@
+(use ../prelude)
+(import ../core)
+(use ./capstone)
+(use ./file)
+(use ./xxhash)
+(use ./zlib)
+(use ./libzip)
+(use ./lz4)
+(use ./openssl)
+(use ./libuv)
+
+# XXX tree-sitter can be used to enable features.
+
+(defsrc radare2-src
+  :file-name
+  "radare2-4.4.0.tar.gz"
+  :url
+  "https://github.com/radareorg/radare2/archive/4.4.0.tar.gz"
+  :hash
+  "sha256:401ab89f39f7e70e03336f6890dc4fdf52dae4f61201e4cd8a1fbe5a4b9eb451")
+
+(defpkg radare2
+  :builder
+  (fn []
+    (os/setenv "PKG_CONFIG_PATH"
+               (string
+                (join-pkg-paths ":" "/lib64/pkgconfig"
+                                [libuv libzip])
+                ":"
+                (join-pkg-paths ":" "/lib/pkgconfig"
+                                [capstone zlib openssl xxhash lz4])))
+
+    (os/setenv "PATH"
+               (join-pkg-paths ":" "/bin"
+                               [core/coreutils
+                                core/findutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/patch
+                                core/sed
+                                core/awk
+                                core/tar
+                                core/pkgconf
+                               ]))
+    (core/link-/bin/sh)
+    (unpack-src radare2-src)
+    (os/setenv "CFLAGS"
+               (string *default-cflags*
+                       " "
+                       (string/join (map |(string "-I" ($ :path) "/include")
+                                         [capstone
+                                          file
+                                          zlib
+                                          lz4
+                                          xxhash
+                                          openssl
+                                          libuv
+                                          libzip]) " ")
+                       " "
+                       (sh/$<_ pkg-config --cflags capstone "liblz4" zlib libuv libxxhash openssl)))
```

Comment:
Is there a reason for liblz4 being in double quotes?

Reply:
Line 88 has a similar instance...we really like liblz4 for being so fast :)

Reply:
I guess this warrants a comment, but what I observed was that the `.pc` file for `lz4` is called `liblz4.pc` so for `pkg-config` to find the configuration, the argument in the invocation has to have that name. E.g. `/hpkg/345928850d3e62e09f254d4b68f2dbd119300901-lz4/lib/pkgconfig/liblz4.pc`

Reply:
ah, but it need not be in quotes despite that, right?

Reply:
Yes, I think that is correct.

Reply:
quotes removed, build works